test-data/unit/check-python2.test

```diff
@@ -13,14 +13,6 @@ s = b'foo'
 from typing import TypeVar
 T = TypeVar(u'T')
 
-[case testNamedTupleUnicode]
```

Comment:
This no longer worked because my stub for namedtuple just has str. Apparently we don't support `Text` in the test stubs, so I just moved this test case to the pythoneval tests.